CombinatorialGames/IGame/Concrete.lean

```diff
@@ -71,6 +72,20 @@ theorem leftMoves_toIGame (a : α) : (toIGame a).leftMoves = toIGame '' {b | b 
 theorem rightMoves_toIGame (a : α) : (toIGame a).rightMoves = toIGame '' {b | b ≺ᵣ a} := by
   rw [toIGame_def, rightMoves_ofSets]
 
+theorem rel_leftMove_iff {a b : α} :
+    toIGame b ∈ (toIGame a).leftMoves ↔ ∃ c, c ≺ₗ a ∧ toIGame c = toIGame b := by
+  simp_rw [leftMoves_toIGame, Set.mem_image, Set.mem_setOf_eq]
+
+theorem rel_leftMove {a b : α} (hab : b ≺ₗ a) : toIGame b ∈ (toIGame a).leftMoves :=
```

Comment:
```suggestion
theorem mem_leftMoves_toIGame_of_relLeft {a b : α} (hab : b ≺ₗ a) : toIGame b ∈ (toIGame a).leftMoves :=
```
Somewhat wordy name but it matches e.g. `NatOrdinal.mem_leftMoves_toIGame_of_lt`.